app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
+        ):
+            file_out.write(file_in.read())
+        os.remove(command[1])
+        return
+    for directory in route:
+        if not os.path.exists(directory):
```

Comment:
Before creating a directory, you should check if it already exists to avoid errors. You can use `os.path.exists(directory)` for this check【4:2†checklist.md】.